Wikipedia/Code/WMFAppViewController+Extensions.swift

```diff
@@ -614,7 +614,11 @@ extension WMFAppViewController {
         Task {
             do {
                 let yirDataController = try WMFYearInReviewDataController()
-                try await yirDataController.populateYearInReviewReportData(for: year, countryCode: countryCode, primaryAppLanguageProject: project)
+                try await yirDataController.populateYearInReviewReportData(
+                    for: year,
+                    countryCode: countryCode,
+                    primaryAppLanguageProject: project,
+                    username: dataStore.authenticationManager.authStatePermanentUsername ?? "")
```

Comment:
See other comments - lets allow an optional username to be passed through instead of the `?? ""`